pkgs/misc/lssecret/default.nix

```
@@ -6,12 +6,12 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "lssecret";
+  pname = "lssecret";
   version = "unstable-2022-12-02";
 
   src = fetchFromGitLab {
     owner = "GrantMoyer";
-    repo = name;
+    repo = pname;
```

I would not do that. I would replace `fetchFromGitLab.pname` with a literal. 
```suggestion
    repo = "lssecret";
```